### PR TITLE
fix: remove extra deps

### PR DESCRIPTION
**Pull request by achingbrain**
Remove extra/add missing deps shown by https://github.com/ipfs/aegir/pull/1426/

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/utils/package.json [hunk 1]
```diff
@@ -112,28 +112,26 @@
   "dependencies": {
     "@chainsafe/is-ip": "^2.0.2",
     "@libp2p/interface": "^1.1.0",
-    "@libp2p/peer-collections": "^5.1.2",
```
Comment (maschad) [inline packages/utils/package.json:115]: Why was this dependency removed? It's used in the [peer-job-queue](https://github.com/libp2p/js-libp2p/blob/main/packages/utils/src/peer-job-queue.ts#L4)

**Review by maschad — CHANGES_REQUESTED** [review]
Perhaps we should wait till https://github.com/ipfs/aegir/pull/1426 is merged and aegir is upgraded so that we can re-run the _fixed_ `dep-check` on CI before merging.

File: packages/utils/package.json (hunk 1 above)
Comment (achingbrain) [reply packages/utils/package.json]: It was removed in #2339, this PR depended on that one, I thought I'd mentioned that in the description but I guess I missed it..

**Comment by achingbrain** [comment]
Aegir is upgraded.

**Pull request merged**